README.md

```diff
@@ -39,8 +39,8 @@ To get a local copy up and running follow these simple example steps.
 
 1. ``` git clone https://github.com/evansinho/React-Bookstore.git ```
 2. ``` cd React-Bookstore ```
-3. ``` git checkout  start```
-4. ``` yarn i ```
+3. ``` git checkout  milestone5```
+4. ``` npm i ```
```

Comment:
Don't you think this instruction will be outdated after merging to your base (default) branch?

I would recommend removing the `git checkout  milestone5` option, this means when this branch is merged to your base branch, there won't be a need for you to checkout as the needed files would be available on the base branch.